### PR TITLE
Normalize language server installer file paths

### PR DESCRIPTION
**Pull request by aeschright**
Fixes problems on some Windows environments

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -260,7 +261,7 @@ async function pathToBinary(): Promise<string> {
 	if (!_pathToBinaryPromise) {
 		let command: string = config('terraform').get('languageServer.pathToBinary');
 		if (!command) { // Skip install/upgrade if user has set custom binary path
-			const installDir = `${extensionPath}/lsp`;
+			const installDir = path.join(extensionPath, '/lsp');
```
Comment (radeksimko) [inline src/extension.ts:264]: The documentation doesn't seem very clear on how it handles such case (except that it "allows" it), but shouldn't the platform-dependent separators be now omitted? i.e. shouldn't `/lsp` be just `lsp`?

File: src/extension.ts [hunk 2]
```diff
@@ -270,7 +271,7 @@ async function pathToBinary(): Promise<string> {
 			} finally {
 				await installer.cleanupZips(installDir);
 			}
-			command = `${installDir}/terraform-ls`;
+			command = path.join(installDir, '/terraform-ls');
```
Comment (radeksimko) [inline src/extension.ts:274]: Same question here ^

**Review by radeksimko — COMMENTED** [review]
This makes sense in principle, I'm just not sure about how `path.join` actually works when there is a platform-dependent separator inside of the path segment.

Generally though if it doesn't need to be there, it probably shouldn't?

File: src/extension.ts (hunk 1 above)
Comment (aeschright) [reply src/extension.ts]: My impression was that it would normalize the separator as well, but it works without the slash so I'll go with that.

**Review by radeksimko — APPROVED** [review]

**Pull request merged**